sp/src/game/server/hl2/item_dynamic_resupply.cpp

```diff
@@ -303,7 +306,9 @@ void CItem_DynamicResupply::Precache( void )
 	for ( i = 0; i < NUM_HEALTH_ITEMS; i++ )
 	{
 #ifdef EZ
-		UTIL_PrecacheEZVariant( g_DynamicResupplyHealthItems[i].sEntityName, m_tEzVariant );
+		// Don't precache unless we might spawn this
+		if (m_flDesiredHealth[i] > 0.0f)
```

Comment:
Shouldn't this be "Desired Ammo"?

Reply:
This loop is precaching the desired health, not the desired ammo. The loop precaching desired ammo is directly below this change.

This change to the health item loop is not necessary to prevent Xen being precached when it shouldn't be, but the fix for the latter applies to all ammo types and I figured I would do the same for the health items since it's easy and would allow them both to follow the same new standard.

Reply:
I understand now, that was my mistake.